lambdas/services/base/s3_service.py

```diff
@@ -12,41 +13,49 @@
 class S3Service:
     _instance = None
 
-    def __new__(cls):
+    def __new__(cls, *args, **kwargs):
```

Comment:
Any reason for the args/kwargs addition?

Reply:
`__new__` is run before `__init__` and all the args are passed into both. So I need to be able to pass the custom role into `__init__`

Reply:
When __new__() returns an instance of cls it automatically invokes the [__init__()] method of that instance with the arguments passed to it.(https://www.codecademy.com/resources/docs/python/dunder-methods/new)